packages/Loader/src/Spinner.tsx

```diff
@@ -0,0 +1,63 @@
+import withUtils from "@blaze-react/utils";
+import React from "react";
+interface ISpinnerCustomStyles {
+  size?: number;
+  backgroundColor?: string;
+  border?: string;
+  borderTopColor?: string;
+}
+interface ISpinnerProps {
+  customStyles: ISpinnerCustomStyles;
+  utils: {
+    classNames: (className: string | object, classNames?: object) => string;
+  };
+  lockContent?: boolean;
+  animation: "ease" | "linear";
+}
+
+const Spinner = withUtils(
+  ({
+    customStyles: { size, backgroundColor, border, borderTopColor },
```

Comment:
not sure about have this kind of props, styles I thiink should be in modifiers and set in css